tests/FixedArrayTest.php

```diff
@@ -166,6 +166,65 @@ public function providerShiftDown() : array
         ];
     }
 
+    public function testCreate()
+    {
+        $fixedArray = FixedArray::fromArray([1, 2, 3, 4]);
+        $result = $fixedArray::create();
```

Comment:
This should be `FixedArray::create()`. The line above can be deleted.

Reply:
You're right, I will use the ```FixedArray::create()``` to replace the previous syntax.